docs/en/stack/ml/nlp/ml-nlp-extract-info.asciidoc

```diff
@@ -100,4 +101,40 @@ The task returns the following result:
 }
 ...
 ----------------------------------
+// NOTCONSOLE
+
+
+[discrete]
+[[ml-nlp-question-answering]]
+== Question answering
+
+The question answering (or extractive question answering) task makes it possible 
+to get answers to certain questions by extracting information from the provided 
+text.
+
+The model tokenizes the string of – usually long – unstructured text, then it 
+attempts to pull the answer from the text to the provided question. The logic is 
+shown by the following examples:
+
+[source,js]
+----------------------------------
+...
+{
+    "text_field": "The Amazon rainforest (Portuguese: Floresta Amazônica or Amazônia; Spanish: Selva Amazónica, Amazonía or usually Amazonia; French: Forêt amazonienne; Dutch: Amazoneregenwoud), also known in English as Amazonia or the Amazon Jungle, is a moist broadleaf forest that covers most of the Amazon basin of South America. This basin encompasses 7,000,000 square kilometres (2,700,000 sq mi), of which 5,500,000 square kilometres (2,100,000 sq mi) are covered by the rainforest. This region includes territory belonging to nine nations. The majority of the forest is contained within Brazil, with 60% of the rainforest, followed by Peru with 13%, Colombia with 10%, and with minor amounts in Venezuela, Ecuador, Bolivia, Guyana, Suriname and French Guiana. States or departments in four nations contain "Amazonas" in their names. The Amazon represents over half of the planet's remaining rainforests, and comprises the largest and most biodiverse tract of tropical rainforest in the world, with an estimated 390 billion individual trees divided into 16,000 species.",
+    "question": "Which name is also used to describe the Amazon rainforest in English?"
+}
```

Comment:
Should this look like the `_infer` API? If so, we need to make some changes...

Reply:
@benwtrent 
On this page, we use pseudo-examples to make the logic of the tasks more understandable but without the low-level details of the API calls. This is why these examples are marked as `//NOTCONSOLE`, so the rendered page won't urge users to copy/paste the examples.

Reply:
ok. 

```
"docs": [{"text_field": <long_text>}],
"inference_config": {"question_answering": {"question": <question>}}
```
Is the correct syntax for what its worth